resources/views/objects/mobile-header.blade.php

```diff
@@ -0,0 +1,125 @@
+                            style="background-image: url('{{ Auth::user()->user_avatar }}');"
+                        >
+                        </a>
+                    @else
+                        <a
+                            href="#"
+                            data-toggle="modal"
+                            data-target="#user-dropdown-modal"
+                            title="{{ trans('users.anonymous.login_link') }}"
+                            class="avatar avatar--navbar-mobile avatar--guest js-navbar-mobile--top-icon"
+                        >
+                        </a>
+                    @endif
+
+                    <button
+                        type="button"
+                        class="navbar-toggle navbar-mobile__toggle colour-hover--{{ $current_section }}"
+                        data-toggle="collapse" data-target="#xs-navbar"
+                    >
+                        <span class="sr-only">Toggle navigation</span>
+                        <div class="navbar-mobile__toggle-icon">
+                            <i class="fa fa-bars"></i>
+                        </div>
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="collapse navbar-collapse navbar-mobile__menu js-navbar-mobile--menu" id="xs-navbar">
+        <ul class="nav navbar-nav navbar-mobile__menu-items">
+            <li class="navbar-mobile__user">
+                @if (Auth::check())
+                    <a class="navbar-mobile__menu-item navbar-mobile__menu-item--user" href="{{ route('users.show', Auth::user()) }}">
+                        <span
+                            class="avatar avatar--navbar-mobile"
+                            style="background-image: url('{{ Auth::user()->user_avatar }}');">
+                        </span>
+
+                        {{ Auth::user()->username }}
+                    </a>
+
+                    <a
+                        class="navbar-mobile__menu-item navbar-mobile__menu-item--logout js-logout-link"
+                        href="{{ route('users.logout') }}"
+                        data-method="delete"
+                        data-confirm="{{ trans('users.logout_confirm') }}"
+                        data-remote="1"
+                    >
+                        <i class="fa fa-sign-out"></i>
+                    </a>
+                @else
+                    <a class="navbar-mobile__menu-item navbar-mobile__menu-item--user" href="#" title="{{ trans('users.anonymous.login_link') }}" data-toggle="modal" data-target="#user-dropdown-modal">
+                        <span class="avatar avatar--guest avatar--navbar-mobile"></span>
+
+                        {{ trans('users.anonymous.username') }}
+                    </a>
+                @endif
+            </li>
+            @foreach (nav_links() as $section => $links)
+            @if(isset($subLinks) && !$subLinks)

```

Comment:
The menu on desktop doesn't have sublinks, so I need the mobile version to reflect it